apps/roam/src/utils/types.ts

```diff
@@ -46,3 +46,14 @@ export type Result = {
   Record<string, string | number | Date>;
 
 export type Column = { key: string; uid: string; selection: string };
+
+export type DiscourseGraphContent = {
```

Comment:
Let's collocate this in the function for now.  Will probably be replaced by Zod.  We can move it to types if we need to reuse it.